src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1154,6 +1167,8 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             return;
         }
 
+        _pointerPressedInBounds = false;

```

Comment:
And because we use _capture_... if the cursor is pressed inside and released outside, we still get this message and clear the flag yes?

Reply:
🤔

Reply:
Checked out the branch and confirmed - yes we do